test/loader/platforms/platforms.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ *
+ * Copyright (C) 2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ */
+#include <cassert>
+#include <cstdlib>
+#include <iostream>
+#include <logger/ur_logger.hpp>
```

Comment:
nit: Please, make two separate groups of headers: one group with headers from the standard library and the other with UR headers.